test_utils/tasks.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tasks used in tests
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from test_utils.celery import app
+
+from celeryutils import persist_on_failure
+
+
+@app.task(base=persist_on_failure.PersistOnFailureTask)
+def fallible_task(message=None):
+    """
+    Simple task to let us test retry functionality.
+    """
+    if message:
+        raise ValueError(message)
+
+
+@app.task(base=persist_on_failure.PersistOnFailureTask)
+def passing_task():
+    """
+    This task always passes
+    """
+    return 5
```

Comment:
Shouldn't the body of this task method look like this?
```
def passing_task():
     """
     This task always passes
     """
     pass
```


![image](https://cloud.githubusercontent.com/assets/7373924/22570265/a861d374-e968-11e6-85db-3e0298c47305.png)